src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        Colors[] colors = Colors.values();
+        Random random = new Random();
+        int index = random.nextInt(colors.length);
+        return String.valueOf(colors[index]);
```

Comment:
As described in common mistakes, it's better to use name() method in order to get string representation of enum